modules/combined/tests/elastic_thermal_patch/elastic_thermal_jacobian_rz_smp_test.i

```diff
@@ -75,76 +71,66 @@
     order = CONSTANT
     family = MONOMIAL
   [../]
-
-[] # AuxVariables
-
-[SolidMechanics]
-  [./solid]
-    disp_r = disp_x
-    disp_z = disp_y
-    temp = temp
-  [../]
 []
 
 [Kernels]
-
-#  [./body]
-#    type = BodyForce
-#    variable = disp_y
-#    value = 1
-#    function = body
-#  [../]
+  [./TensorMechanics]
+    displacements = 'disp_x disp_y'
```

Comment:
To avoid setting displacements for the mesh block which is the case for counterpart sm test, otherwise the test will failure when compared with solid mechanics results.